src/signals/incident-management/containers/FilterTagList/__tests__/FilterTagList.test.js

```diff
@@ -164,6 +164,8 @@ describe('signals/incident-management/containers/FilterTagList', () => {
     it('works with feature flag fetchDistrictsFromBackend enabled', () => {
       configuration.fetchDistrictsFromBackend = true;
 
+      // stadsdeel is intentionaly removed from the tags list here
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
It can be written as `const { , ...otherTags } = tags;` then, can't it?

Reply:
I tried that but the linter doesn't like it. Let's solve this later.